frontend/public/components/ingress.jsx

```diff
@@ -103,10 +103,18 @@ const RulesHeader = () => {
   const { t } = useTranslation();
   return (
     <div className="row co-m-table-grid__head">
-      <div className="col-xs-3">{t('ingress~Host')}</div>
-      <div className="col-xs-3">{t('ingress~Path')}</div>
-      <div className="col-xs-3">{t('ingress~Service')}</div>
-      <div className="col-xs-2">{t('ingress~Service port')}</div>
+      <div data-i18n-translated className="col-xs-3">
```

Comment:
It would be better to match all table headers IMO instead of using `data-i18n-translated`. We can consider adding a data attribute for things we expect *not* to be translated, but we should assume by default everything is translated. You should have to opt out of the tests, not opt in.

Reply:
I was able to reduce the number of `data-i18n-translated` by using the `dt` selector, however there are still a few examples of translated text in various `div`'s, `span`'s, and `p`'s;  I'm not sure how to programatically select, or omit, those to only isolate those examples.

Reply:
In other words, how can you tell if a `div` is showing a transalted label or data?

Reply:
I would avoid trying to test every string on the page and focus on the common elements that should be translated for all pages. These tests I think are most valuable to catch new strings that weren't properly externalized.